.github/workflows/post-outdated-content-report.yaml

```diff
@@ -103,12 +105,12 @@ jobs:
         
         echo "" >> report.md
         
-        echo "### Details" >> report.md
+        echo "### Changes in each file" >> report.md
         
         for FILE in ${FILE_LIST}; do 
           FILE_NAME="${FILE#${OUTPUT_DIR}}" 
 
-          echo "Outdated file: ${FILE_NAME}" >> report.md
+          echo "${FILE_NAME}" >> report.md
```

Comment:
Task 처리 여부 파악 및 PR에 Task수 노출을 위해서
`- [ ]` 를 추가하면 어떨까요? :)

Reply:
관련 파일의 저장소상 url을 제공해주면 어떨까요? 
바로 클릭해볼 수 있으면, 조금 더 적극적으로 반응하여 기여할 수 있을 것 같습니다. ㅎㅎ
```
- [ ] container-as-a-service.md
- en: https://github.com/cncf/glossary/blob/main/content/en/containers-as-a-service.md
- ko: https://github.com/cncf/glossary/blob/dev-ko/content/ko/containers-as-a-service.md
```
이런 식으로..?

- [ ] container-as-a-service.md
- en: https://github.com/cncf/glossary/blob/main/content/en/containers-as-a-service.md
- ko: https://github.com/cncf/glossary/blob/dev-ko/content/ko/containers-as-a-service.md

Reply:
상세 내용은 hidden 기능을 활용해서, 이슈를 깔끔하게 만들면 어떨까요?

<details>
<summary><b>Diff in detail</b></summary>

```diff
diff --git a/content/en/contribute/_index.md b/content/en/contribute/_index.md
index 1fab015..f788ad7 100644
--- a/content/en/contribute/_index.md
+++ b/content/en/contribute/_index.md
@@ -7,74 +7,104 @@ menu:
     weight: 10
 ---

-The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
-where you'll find a list of [issues](https://github.com/cncf/glossary/issues), [PRs](https://github.com/cncf/glossary/pulls), and 
-[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
+## Welcome 
 
-There are four ways you can contribute:
+Welcome to the Cloud Native Glossary contributing guide, and thank you for your interest. 
+There are a number of ways you can contribute to this project, which we'll cover in detail here: 
 
 1) [Work on an existing issue](#work-on-an-existing-issue)
 2) [Propose new terms](#propose-new-terms)
 3) [Update existing ones](#update-an-existing-term)
-4) [Help translate the glossary](#help-translate-the-glossary)
+4) [Localize the glossary](#help-localize-the-glossary)
+
+## CNCF glossary overview 
+
+The goal of this glossary is to simplify the cloud native space — which is notorious for its complexity — and thus make it more accessible to people. 
+
+The Cloud Native Glossary content is stored in [this GitHub repo](https://github.com/cncf/glossary) 
+where you'll find a list of [issues](https://github.com/cncf/glossary/issues), pull requests ([PRs](https://github.com/cncf/glossary/pulls)), and 
+[discussions](https://github.com/cncf/glossary/discussions) about the glossary. 
 
 ## Who can contribute?
 
-The goal of this Glossary is to make the cloud native space — which is notorious for its complexity — simpler for people by making it easier to understand. 
-To do that, we must focus on simplicity. 
+How you can participate in this project depends on your level of cloud native expertise. 
+Simplifying complex concepts requires a deep knowledge of the topic. 
+Therefore, to contribute new terms, you must be proficient in them. 
+Contributors are typically engineers who have worked with these technologies for some time or academics focused on cloud native. 
+
+That know-how is required because explaining complex concepts in simple words is _really_ hard. And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity results from hard work and collaboration between cloud native experts. 
+
+If you are not a cloud native expert yet but still want to contribute, we recommend teaming up with someone who is. 
+Once the expert is confident that the term accurately describes the concept, you are ready for your first Glossary contribution.
+
+The localization effort is where beginners proficient in another language can make valuable contributions to the Glossary. 
+With solid existing definitions in English, less experienced contributors can localize terms to a target language. You can join an existing localization team or create a new one. Read this guide's [Help Localize the glossary](#help-localize-the-glossary) section to learn how to get started. 
+
+## Before you start
+
+Before beginning your Glossary contributor journey, be sure to complete the following steps:
+
+1. Create a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account), if you don't have one already. 
+2. Sign the [Contributor License Agreement](https://docs.linuxfoundation.org/lfx/easycla/v2-current/contributors) (CLA). 
+3. [Verify your commit signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification)
+4. Enable [vigilant mode](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits#about-vigilant-mode) in your GitHub account to display the "Verified" status on your commits. 
 
-Explaining complex concepts in simple words is *really* hard. 
-And while the digestible, user-friendly outcome may seem easy, achieving the desired simplicity is the result of hard work and collaboration between cloud native experts.
-In short, to contribute new terms, you need a deep understanding of the topic. 
+## Best practices {#best-practices}
 
-The localization effort is a bit different. 
-With solid existing definitions in English, less experienced contributors can still provide good localized terms. 
-This side of the project is better suited for beginners.
+To facilitate the reviewing process, please use [semantic line breaks](https://sembr.org/) (e.g., one line per sentence).
+We recommend checking out this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/) 
+to correctly format Markdown text in GitHub (e.g., hyperlink, bold, italic).
+And when naming .md files, please use lowercase letters and hyphens instead of spaces to separate words and avoid parenthesis.
+
+## Style guide
+
+Read our [Style Guide](/style-guide/) to understand our guidelines for formatting and writing documents and make the contribution process more efficient. 
 
 ## Join the Glossary community! {#join-the-glossary-community}
 
-Consider joining our monthly Glossary Working Group meetings if you want to contribute regularly. 
-You can find meeting details in the [CNCF calendar](https://www.cncf.io/calendar/). 
+If you want to contribute regularly, consider joining our monthly Glossary Working Group meetings. 
+You can find the meeting details in the [CNCF calendar](https://www.cncf.io/calendar/). 
 You can also connect with the maintainers and fellow contributors in our [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel on the CNCF Slack 
 — we'd love to meet you! 
 
 ## Work on an existing issue {#work-on-an-existing-issue}
 
-Go to the [Glossary GitHub repo issues](https://github.com/cncf/glossary/issues). 
-There you'll see a list of all issues. You can filter by label (e.g. English language, help needed, good first issue). 
-Note that you'll need a GitHub account to do any of this.
+Go to the [Glossary GitHub repo issues](https://github.com/cncf/glossary/issues) to find a list of available issues. 
+You can use labels (e.g., English language, help needed, good first issue) to filter out issues.
 
 ![Issue and labels](/images/how-to/issue-and-labels.png)
 
-Make sure the term you are interested in is not already assigned to someone. 
-Here you can see that the first three terms are available while the next term has already been assigned.
+Be sure the term you select hasn't already been assigned to anybody. 
+For example, here you can see that the first three terms are available while the fourth one has already been assigned.
 
 ![assigning a term](/images/how-to/howto-04.png)
 
-Once you found a term you want to work on, say so in the issue. Click on it and add a comment.
+Once you select a term to work on, comment on the issue. 
 
 ![Claiming an issue](/images/how-to/claiming-an-issue.png)
 
-Additionally, please also join the [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel on the CNCF Slack and 
-let the maintainers know that you've raised an issue for a new term 
-(ideally, tag _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ to be sure they don't miss it). 
-Note that you can only claim one term at a time. 
-If you want to work on multiple terms, please finish one before claiming the next one.
+Additionally, notify the maintainers on the [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) channel of the CNCF Slack workspace and 
+tag _@Catherine Paganini_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ to be sure they don't miss it. 
 
-Once they assign it to you, you can start working on it. 
 For the next steps, please refer to the [Submitting a new term (creating a PR)](#submitting-a-new-term) section.
 
+**Note**: you can start working on an issue after the maintainers assigned it to you. 
+You can only claim one term at a time. 
+Workig on multiple terms is sequential, you must complete a term before claiming the next one.
+
 ## Propose new terms {#propose-new-terms}
 
 You can propose a new term for others to work on or create a new definition yourself. 
-Either way, you'll start by creating an issue. 
-Please note that terms must meet the [CNCF's cloud native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md). 
+Either way, you'll start by [creating an issue](#creating-an-issue). 
+To be added to the glossary, every new term must meet the [CNCF's cloud native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md). 
 The only exceptions are foundational terms needed to understand cloud native concepts.
 
-Below is a step-by-step guide for those not yet familiar with GitHub. 
-**If you are a GitHub Pro**, please _do_ have a quick look to make sure you use our issue templates, 
-appropriate naming conventions, claim a PR on Slack (otherwise we may miss it), and where to find the file template. 
-And please make sure to read the [Style Guide](/style-guide/) before getting started — thank you! 
+Below is a step-by-step guide for people unfamiliar with GitHub. 
+**If you are a GitHub Pro**, please _do_ scan this guide to gather enough information about the following topics:
+
+1. Locating templates for issues and new terms.
+2. Claiming issues. 
+3. Solving [spell check](#spell-check) failures. 
 
 ### Creating an issue {#creating-an-issue}
 
@@ -82,36 +112,31 @@ Go to the [Glossary GitHub repo](https://github.com/cncf/glossary/issues) issues
 
 ![issues](/images/how-to/howto-01.png)
 
-You'll see a variety of templates. To propose a new term in English, select "Request to add a new term (English)".
+Select "Request to add a new term (English)" from the list of templates. 
 
 ![templates](/images/how-to/english-issue-template.jpg)
 
-Add the word you're suggesting, answer the two questions below, check the checkboxes, and hit "Submit new issue".
-If you're just proposing a new term, you're done! To work on it, follow the next steps.
+Add the word you're suggesting, answer the questions, check the boxes, and hit "Submit new issue".
+If you're just proposing a new term, you're done! If you want to work on the definition, keep reading.
 
 ### Triaging your issue {#triaging-your-issue}
 
 Next, the maintainers will triage the issue. 
-That means they will assess if the term should be part of the Glossary 
-(note, not every term will be admitted. Terms should be established and widely-used cloud native terms).
+That means they will assess if the term should be part of the Glossary. 
+Not every term will be admitted. To be included in the Glossary, they should be established and widely-used cloud native concepts.
 
-Please let the maintainers know that you've submitted a term on Slack as they may otherwise miss it. 
-Ideally, tag _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_. 
-If the term is approved and you want to work on it, they'll assign it to you.
-
-Note that you can only claim one term at a time. 
-If you want to work on multiple terms, please finish one before claiming the next.
+Please let the maintainers know that you've proposed a new term on Slack and tag _@Catherine Paganini_, _@Seokho Son_, _@Jihoon Seo_, and/or _@iamnoah_ so that hey don't miss it.  
+If you want to work on the definition, let the maintainers know and they'll assign it to you.
 
 ### Submitting a new term (creating a PR) {#submitting-a-new-term}
 
-Before getting started, please read the [Style Guide](/style-guide/) — it will help minimize backs and forth. 
-As stated in the style guide, we highly recommend starting with a Google or Word doc. 
+As stated in our [Style Guide](/style-guide/), we highly recommend starting with a Google or Word doc. 
 
-Once the term is ready to submit, go to content (under code)…
+Once the term is ready to submit, go to content (under <>code)…
 
 ![content](/images/how-to/howto-05.png)
 
-…then "en" (or the language you are submitting for)…
+…then "en" (for English) or the first two letters of the language you are contributing to…
 
 ![language folder](/images/how-to/howto-06.png)
 
@@ -123,61 +148,78 @@ Copy the content…
 
 ![copy content](/images/how-to/howto-08.png)
 
-…and go back to the "en" folder. Hit "Add file" and select "Create new file".
+…and go back to the "en" folder. Click "Add file" and select "Create new file".
 
 ![create new file](/images/how-to/howto-09.png)
 
-Add the term name in the URL (no capitalization, no spaces, no parentheses, and use hyphens to separate words)
-and .md at the end (note: if your preview doesn't work, you probably forgot to add .md at the end).
-Now paste the template content below. Copy and paste your definition into the file.
-To make reviews easier, please **use [semantic line breaks](https://sembr.org/)** (e.g. one line per sentence).
-Note that GitHub uses markdown to format the text (e.g., hyperlink, bold, italic).
-Please refer to this [markdown cheat sheet](https://www.markdownguide.org/cheat-sheet/).
-To verify you've used markdown as intended, go to "Preview".
+Add the name of the term in the URL as described in the [Best practices](#best-practices) section. 
+Add the .md extension at the end of the name (without this extension you won't be able to preview your file).
+Now paste the template content in the section below. Copy and paste the text of your definition into the file.
+To verify you've used Markdown as described in the [Best practices](#best-practices) section, click on "Preview".
 
 ![finalize term](/images/how-to/howto-10.png)
 
-Scroll down and name the new commit file when you are ready to submit. 
-You are now about to commit the term to your own branch. 
-Submitting a PR requires one more step. Hit "Commit new file" and…
+Scroll down and name the new commit file. Hit "Commit new file" when you are ready to submit 
+and…
 
 ![commit new file](/images/how-to/howto-11.png)
 
-…now you're creating an PR:
+… you're now ready to create a new PR:
 
 ![create a pr](/images/how-to/howto-12.png)
 
-You should now see your PR under "Pull requests".
+Once you hit the "Create pull request" button your PR should be visible in the "Pull requests" tab.
 
 ![prs](/images/how-to/howto-13.png)
 
 ## Update an existing term {#update-an-existing-term}
 
-To update an existing term, you can either suggest a change via an issue or go ahead and update the term directly by submitting a pull request (PR).
+To update an existing term, you can either request changes by creating an issue 
+or work on the changes yourself and submit a PR.
 
 ### Request a change via an issue {#request-a-change-via-an-issue}
 
-If you want to flag a problem with a term but don't know how or want to fix it yourself, click on "Report issue".
+If you want to flag a problem with a term, you can use the "Report issue" option of the CNCF Glossary webpage. 
+Locate yourself in the CNCF page of the concept you'd like to flag and click on "Report issue". 
+This will automatically open an issue for you
 
 ![report issue](/images/how-to/howto-14.png)
 
-This will directly open an issue. Please elaborate on which change is needed and why. Hit submit, and that's it. 
+Please describe your suggestions and why they are needed. Hit submit, and that's it. 
 
 ![submit issue](/images/how-to/howto-15.png)
 
 ### Update a term directly {#update-a-term-directly}
 
-To change a term directly, go to "Edit this page".
+To modify a term and submit your suggestions, click "Edit this page."
 
 ![edit this page](/images/how-to/howto-16.png)
 
-This will open the term's GitHub page. Make your changes and submit a PR. 
-Please refer to "Submitting a new term" above for a detailed description (jump to the section that speaks about markdown).
+This will open the term's GitHub page. Make your changes and create a PR. 
+Please refer to the [Best practices](#best-practices) section above 
+and read our [Style Guide](/style-guide/) to make sure you follow our guidelines. 
 
-## Help translate the glossary {#help-translate-the-glossary}
+## Help localize the glossary {#help-localize-the-glossary}
 
-To help translate the glossary into your native language, please join the [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) channel on the CNCF Slack and let us know.
+To help localize the glossary into a target language, please join the [#glossary-localizations](https://cloud-native.slack.com/archives/C02N2RGFXDF) channel on the CNCF Slack workspace and send us a message.
 You can either join an existing team or create a new one 
-(to see what it takes, check out or [Localization Guide](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md)). 
-Please also join our monthly Glossary Working Group meetings. You can find meeting details in the [CNCF calendar](https://www.cncf.io/calendar/). 
-We look forward to meeting you there!
+(to see what it takes, read our [Localization Guide](https://github.com/cncf/glossary/blob/main/LOCALIZATION.md)). 
+Please read the **How to contribute** guide of the target language to gather the specifics of that team's contribution process. 
+
+## Spell check {#spell-check}
+
+There are two main reasons why the spell check can fail:
+
+- The PR contains misspellings. 
+- The PR contains words that are not registered in the word list. 
+
+To add new words to the list, follow these steps:
+
+1. In your PR, locate the file "wordlist.txt". 
+2. Click "Edit this file" and add the missing words in alphabetic order. 
+3. Add a commit message and select "Sign off and propose changes".
+
+**Note**: the spell check is case-insensitive. 
+
+
+**We updated this guide based on templates from [The Good Docs Project](https://thegooddocsproject.dev/).**
```

</details>

```
<details>
<summary><b>Diff in detail</b></summary>

long diffffffffffffffffffff .

</details>
```

Reply:
@seokho-son 소중한 의견 감사드립니다 ^^

1. ` - [ ]` 은 바로 반영 가능하고요.
2. URL 추가의 경우에 몇 가지 검토 후에 반영하도록 하겠습니다. 
L10n 팀 코드, 브랜치 명, 어느 시점(latest or specific commit)의 파일인지, 등을 고려해서 적절히 수정할 예정 입니다.
5. Hidden 추가는 쉽게 가능합니다. 그런데 추가하기 전에 다른 분들의 의견을 들어보고 싶네요.
기능을 뺐던 이유를 간단히 설명 드리면요. 이전에 Hidden 기능을 고려했다가, 매 파일마다 클릭해서 보기에 불편하다는 생각이 들어 우선 제외 했었습니다. 또한, 변경점이 많지 않을 것이라는 생각도 들었고요. 많은 의견을 부탁드립니다 ^^

Reply:
@yunkon-kim 내용이 한.. 20라인 넘어가는 경우에만 적용하면 어떨까요? :)

Reply:
좋은 의견이십니다 👍 함께 고려해 보겠습니다.

Reply:
PTAL @seokho-son 

말씀주신 의견을 반영하였습니다.

참고) https://github.com/yunkon-kim/test-on-github-actions/issues

Reply:
@seokho-son 이슈 수정하신 양식에 맞추어 추가 커밋 올리겠습니다 ^^

Reply:
계속 업그레이드되고 있군요! 감사합니다. ^^
꼭 해당 PR에서 모든 것이 완료되어야 할 필요는 없으니, 적당히 완료 콜을 주십시오! ㅎㅎ

Reply:
@seokho-son 완료했습니다 ^^